README.md

```diff
@@ -11,10 +11,14 @@ Refract is a recursive data structure for expressing complex structures, relatio
 
 ## Version
 
-**Current Version**: 0.6.0
+**Current Version**: 0.7.0
```

Comment:
Both this PR and https://github.com/refractproject/refract-spec/pull/52/files#diff-04c6e90faac2675aa89e2176d2eec7d8R14 changes version to 0.7.0.

I think these PRs shouldn't change the version number. Instead once both have been merged, there should be a separate release PR (once we deem ready for release, perhaps other upcoming changes can make it into this release too).